src/network/taraxa_capability.cpp

```diff
@@ -592,31 +607,72 @@ void TaraxaCapability::interpretCapabilityPacketImpl(NodeID const &_nodeID, unsi
       auto pbft_current_round = pbft_mgr_->getPbftRound();
       Vote vote(_r[0].data().toBytes());
       auto peer_pbft_round = vote.getRound() + 1;
-      LOG(log_nf_next_votes_sync_) << "Received " << next_votes_count << " next votes from peer " << _nodeID
-                                   << " node current round " << pbft_current_round << ", peer pbft round "
-                                   << peer_pbft_round;
-
       std::vector<Vote> next_votes;
       for (size_t i = 0; i < next_votes_count; i++) {
         Vote next_vote(_r[i].data().toBytes());
-        LOG(log_nf_next_votes_sync_) << "Received PBFT next vote " << next_vote.getHash();
+        if (next_vote.getRound() != peer_pbft_round - 1) {
+          LOG(log_er_next_votes_sync_) << "Received next votes bundle with unmatched rounds from " << _nodeID
+                                       << ". The peer may be a malicous player, will be disconnected";
+          host->disconnect(_nodeID, p2p::UserReason);
+
+          break;
+        }
```

Comment:
This will always true. because line 609

Reply:
I disagree @gaoqi8649 ,  609 only checks the first vote received relative to node's round.   This line checks that all votes received are in the same round.